.github/workflows/restart_recovery_suite.yml

```diff
@@ -0,0 +1,118 @@
+on:
+  schedule:
+    - cron: '0 5 * * *'
+  push:
+    branches:
+      - '*' # matches every branch
+  pull_request:
+    branches:
+      - '*' # matches every branch
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build
+    timeout-minutes: 600
+    runs-on: ubuntu-18.04
+    outputs:
+      err_output: ${{ steps.prepare_err_artifacts.outputs.test}}
+    strategy:
+      fail-fast: false
+      matrix:
+            compiler:
+                - "CONCORD_BFT_CONTAINER_CC=clang CONCORD_BFT_CONTAINER_CXX=clang++"
+            ci_build_type:
+                - "-DCMAKE_BUILD_TYPE=RELEASE -DBUILD_COMM_TCP_TLS=FALSE"
+                - "-DCMAKE_BUILD_TYPE=RELEASE -DBUILD_COMM_TCP_TLS=TRUE"
+            use_s3_obj_store:
+                - "-DUSE_S3_OBJECT_STORE=ON"
+                - "-DUSE_S3_OBJECT_STORE=OFF"
+    steps:
+        - name: Cleanup pre-installed tools
+          run: |
+            # This is a fix for https://github.com/actions/virtual-environments/issues/1918
+            sudo rm -rf /usr/share/dotnet
+            sudo rm -rf /opt/ghc
+            sudo rm -rf "/usr/local/share/boost"
+            sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+        - name: Checkout
+          uses: actions/checkout@v2
+        - name: Create artifact directory
+          run: mkdir -p ${{ github.workspace }}/artifact
+        #- name: Configure core dump location
+        #  run: |
+            # Uncomment the line below to enable core dump collection
+            # echo "/concord-bft/build/cores/core.%e.%p" | sudo tee /proc/sys/kernel/core_pattern
+            # Uncomment this is you want to login into the running session.
+            # Please note that the build will block on this step.
+            # Refer to https://github.com/marketplace/actions/debugging-with-tmate
+        #- name: Setup tmate session
+        #  uses: mxschmitt/action-tmate@v2
+        - name: Build and test
+          run: |
+              script -q -e -c "make pull"
+              sudo df -h
+              script -q -e -c "make build \
+                              ${{ matrix.compiler}} \
+                              CONCORD_BFT_CMAKE_FLAGS=\"\
+                              ${{ matrix.ci_build_type }} \
+                              -DBUILD_TESTING=ON \
+                              -DENABLE_RESTART_RECOVERY_TESTS=TRUE \
+                              -DBUILD_COMM_TCP_PLAIN=FALSE \
+                              -DCMAKE_CXX_FLAGS_RELEASE='-O3 -g' \
+                              -DUSE_LOG4CPP=TRUE \
+                              -DBUILD_ROCKSDB_STORAGE=TRUE \
+                              ${{ matrix.use_s3_obj_store }} \
+                              -DUSE_OPENTRACING=ON \
+                              -DOMIT_TEST_OUTPUT=OFF\
+                              -DKEEP_APOLLO_LOGS=TRUE\
+                              -DUSE_FAKE_CLOCK_IN_TIME_SERVICE=TRUE\" "\
```

Comment:
@titusen and @HristoStaykov , can you please carefully review these parameters together, and make sure each one of them makes sense in the case of the restart recovery suite?

Reply:
We have double checked that all necessary options are correct. Essentially we build the same way we do in clang's build but activate the restart-recovery suite, which will be the only one that is going to be performed in this run. All looks good.